tests/fixtures/rules/no-unused-svelte-ignore/valid/.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+"use strict"
+
+module.exports = {
+  overrides: [
+    {
+      files: ["style-lang07-input.svelte"],
+      rules: {
+        "prettier/prettier": "off",
```

Comment:
Is this expected or is there anything wrong with prettier's rule? And `svelte` should be added into https://github.com/prettier/eslint-plugin-prettier/blob/master/eslint-plugin-prettier.js#L215-L225 for this plugin.

Reply:
Besides, for `eslint-plugin-svelte3`, it can be added in `https://github.com/prettier/eslint-plugin-prettier/blob/master/eslint-plugin-prettier.js#L190`

Reply:
The linting seems to work, but the format seems to fail. (And the log is output)
I haven't explored this issue deeply, but it's probably because prettier-plugin-svelte doesn't handle unknown langIds well. I disabled the rule as a workaround because the logs were in the way when I checked if lint worked :sweat_smile: 

<img width="962" alt="image" src="https://user-images.githubusercontent.com/16508807/176440109-7d965ab5-8a1f-44f8-9356-f84c2471dc40.png">

https://github.com/ota-meshi/eslint-plugin-svelte/runs/7110654990?check_suite_focus=true

Input:

```svelte
<div class="foo">
  <div class="bar" />
</div>

<!-- svelte-ignore css-unused-selector -->
<style lang="unknown">
  .foo
    height: 20px
    width: 20px
    .foo
      height: 10px
      width: 10px
</style>
```

Reply:
I just added better support for `eslint-plugin-svelte` in [`eslint-plugin-prettier@4.2.0`](https://github.com/prettier/eslint-plugin-prettier/blob/master/CHANGELOG.md#420)!